versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/DatabaseAdapter.java

```diff
@@ -141,6 +142,7 @@ Hash commit(CommitAttempt commitAttempt)
    *     defaults to the named reference's HEAD.
    * @param expectedHead if present, {@code target}'s current HEAD must be equal to this value
    * @param sequenceToTransplant commits in {@code source} to cherry-pick onto {@code targetBranch}
+   * @param updateCommitMetadata Function to allow resetting properties application due to merge.
```

Comment:
Please reword. "resetting properties application" is confusing. `merge` is wrong.